newIDE/app/public/external/jfxr/jfxr-electron-index.html

```diff
@@ -0,0 +1,15 @@
+<html>
+
+<head>
+  <title>GDevelop Sound Effects Editor (Jfxr)</title>
+  <link rel="stylesheet" type="text/css" href="gdide://external/utils/path-editor.css">
+  <link rel="stylesheet" type="text/css" href="jfxr-style.css">
```

Comment:
how come this css doesn't need the prefix `gdide://` ?

Reply:
Good question! I wonder if it does or if it's not useful..

FYI this custom gdide://, which is a custom protocol served by Electron, is only useful in "production", because at this point the app is served from the local disk (file:///...) and not from a server (localhost:3000) anymore. So loading relative files seems not to work.

For safety I used gdide:// everywhere.